plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/generated/GesturesAttributeParserTest.kt

```diff
@@ -252,6 +252,20 @@ class GesturesAttributeParserTest {
     val settings = GesturesAttributeParser.parseGesturesSettings(context, attrs, 1.2f)
     assertEquals(1f, settings.zoomAnimationAmount)
   }
+
+  @Test
+  fun enablePinchToZoomWhenRotatingTestTrue() {
```

Comment:
should we check default value?

Reply:
We have such test in `GesturesAttributeParserDefaultValueTest.kt `